package.json

```diff
@@ -18,7 +18,6 @@
   },
   "bugs": "https://github.com/gsong/es6-jspm-gulp-starter/issues/new",
   "jspm": {
-    "directories": {},
```

Comment:
jspm itself is still evolving. It's probably just due to its own internal spec changing. That's why it's best to not change `package.json` and `config.js` by hand if possible.